code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -217,6 +217,13 @@
 	else
 		to_chat(user, "<span class='notice'>Access Denied</span>")
 
+/obj/structure/closet/crate/secure/AltClick(mob/user)
```

Comment:
please check to see if observers can interact here

Reply:
Observers cant interact with them, just tested it. Pretty sure thats handled at something like the /proc/AltClick level